model/application.go

```diff
@@ -1,53 +1,86 @@
 package model
 
-import "github.com/jinzhu/gorm"
-
-// ApplicationModel for Application
-type ApplicationModel struct {
-	gorm.Model
-	Name           string `json:"name"`
-	CatalogName    string `json:"catalogName"`
-	CatalogVersion string `json:"catalogVersion"`
-	Description    string `json:"description"`
-	Icon           string `json:"icon"`
-	OrganizationId uint   `json:"organizationId"`
-	ClusterID      uint
-	Deployments    []*Deployment `gorm:"foreignkey:ApplicationID" json:"deployments"`
+import (
+	"github.com/banzaicloud/pipeline/config"
+	"time"
+)
+
+var log = config.Logger()
+
+// Application for Application
+type Application struct {
+	ID             uint          `json:"id" gorm:"primary_key"`
+	CreatedAt      time.Time     `json:"createdAt"`
+	UpdatedAt      time.Time     `json:"updatedAt"`
+	DeletedAt      *time.Time    `json:"-" sql:"index"`
+	Name           string        `json:"name"`
+	CatalogName    string        `json:"catalogName"`
+	CatalogVersion string        `json:"catalogVersion"`
+	Description    string        `json:"description"`
+	Icon           string        `json:"icon"`
+	OrganizationId uint          `json:"organizationId"`
```

Comment:
I think `OrganizationId` is redundant here as the application belongs to a cluster and the cluster to an organisation.

Reply:
This is not the case. Someone can start a Cluster and Another can deploy Applications. We can rename this to Owner if its more feasable